templates/frontend/pages/article.tpl

```diff
@@ -20,7 +20,7 @@
 	{if $section}
 		{include file="frontend/components/breadcrumbs_article.tpl" currentTitle=$section->getLocalizedTitle()}
 	{else}
-		{include file="frontend/components/breadcrumbs_article.tpl" currentTitleKey="article.article"}
+		{include file="frontend/components/breadcrumbs_article.tpl" currentTitleKey="common.publishedSubmission"}
```

Comment:
Because this involves a breaking change for templates, I should probably restore the `article.article` locale key. I can then add a new `common.publishedSubmission` key for the stats pages and we can use that going forward. But a breaking change like this should only be in a major version (3.x).

Reply:
It looks like you changed this back in `article.tpl`, but the locale key has still been changed in the locale files. I think wherever you find `common.publishedSubmission`, you need to duplicate it with a key of `article.article` (same with plural).

Reply:
Sorry for the confusion on this one. I've opened a PR that does what I was thinking:

https://github.com/bozana/ojs/pull/8

I know this is a bit ugly with two locale keys saying the same thing. But I don't want to remove a locale key that's commonly used in third-party themes in 3.1.2. We can remove it in 3.2.